lib/presets.js

```diff
@@ -283,6 +284,43 @@ module.exports = {
 
         return callback(error);
       });
+    },
+
+    /**
+     * @summary Update Rust Cargo crate version
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} cwd - current working directory
+     * @param {String} version - version
+     * @param {Function} callback - callback (error)
+     * @returns {null}
+     *
+     * @example
+     * presets.updateVersion.cargo({}, process.cwd(), '1.0.0', (error) => {
+     *   if (error) {
+     *     throw error;
+     *   }
+     * });
+     */
+    cargo: (options, cwd, version, callback) => {
+      const cargoToml = path.join(cwd, 'Cargo.toml');
+      const cleanedVersion = semver.clean(version);
+
+      if (!cleanedVersion) {
+        return callback(new Error(`Invalid version: ${version}`));
+      }
+
+      replace({
+        files: cargoToml,
+        from: /(version\s*=\s*)"(?:0|[1-9]\d*)\.(?:0|[1-9]\d*).(?:0|[1-9]\d*)"/g,
+        to: '$1"' + cleanedVersion + '"'
```

Comment:
Unfortunately it does not support template literals and outputs "${cleanedVersion}" directly to the file.